### DISCUSSION TITLE
Resume terminated jobs

### DISCUSSION THREAD

**Discussion by kshitij-v-mehta** [optuna/optuna#5254] [Q&A]
Hello,
We are planning on using Optuna on our cluster to run multiple trials concurrently. The cluster uses the Slurm job scheduler and our runs will be executed in a job allocation on the system.

If a job times out without the Optuna study having completed, some trials could be terminated midway during execution. When we re-submit the job, is there a way Optuna can first re-run the partially completed trials from the previous job? 

Thanks for your help,
Kshitij Mehta
ORNL

**Comment by nzw0301** [comment]
Yes, for example, optuna with pytorch example is https://github.com/optuna/optuna-examples/blob/main/pytorch/pytorch_checkpoint.py.
